CHANGES_UNRELEASED.md

```diff
@@ -4,6 +4,13 @@
 
 [Full Changelog](https://github.com/mozilla/application-services/compare/v0.52.0...master)
 
+## Sync
+
+### What's fixed
+
+- Rust sync code should be more robust in the face of corrupt meta/global
```

Comment:
grammar nit: changelog entries describe the world as it now is, not as it should be; I suggest "Rust sync code is now more robust...."

Reply:
Ah, I guess I was trying to say "Rust sync code should (now) be", but I'll adjust it as you suggest (tomorrow).